src/parsers/pm_io/psse.jl

```diff
@@ -1273,15 +1273,67 @@ function _psse2pm_facts!(pm_data::Dict, pti_data::Dict, import_all::Bool)
     return
 end
 
-function _psse2pm_storage!(pm_data::Dict, pti_data::Dict, import_all::Bool)
-    @warn "This PSS(R)E parser currently doesn't support Storage data parsing..."
-    pm_data["storage"] = []
-    return
+function _build_switch_breaker_sub_data(
+    branch::Dict,
+    device_type::String,
+    discrete_branch_type::Int,
+    index::Int,
+)
+    sub_data = Dict{String, Any}()
+    sub_data["f_bus"] = branch["I"]
+    sub_data["t_bus"] = branch["J"]
+    sub_data["r"] = branch["R"]
+    sub_data["x"] = branch["X"]
+    sub_data["state"] = branch["ST"]
+    sub_data["active_power_flow"] = 0.0
+    sub_data["reactive_power_flow"] = 0.0
+    sub_data["rating"] = branch["RATEA"]
+    sub_data["discrete_branch_type"] = discrete_branch_type
+    sub_data["name"] = "$(device_type == "breaker" ? "BRK" : "SWT")_$(branch["I"])_$(branch["J"])"
+    sub_data["source_id"] =
+        [device_type, sub_data["f_bus"], sub_data["t_bus"], sub_data["name"]]
+    sub_data["index"] = index
+    return sub_data
 end
 
-function _psse2pm_switch!(pm_data::Dict, pti_data::Dict, import_all::Bool)
-    @warn "This PSS(R)E parser currently doesn't support Switch data parsing..."
+function _psse2pm_switch_breaker!(pm_data::Dict, pti_data::Dict, import_all::Bool)
+    @warn "Parsing PSS(R)E Switches & Breakers data into a PowerModels Dict..."
+    pm_data["breaker"] = []
     pm_data["switch"] = []
+
+    if haskey(pti_data, "BRANCH")
+        for branch in pti_data["BRANCH"]
```

Comment:
We are looping over this dict multiple times. We can `pop!` the entries that are switches before we run it by the "lines"